bower.json

```diff
@@ -52,7 +52,8 @@
     "promise-polyfill": "^5.2.1",
     "font-awesome": "fontawesome#^4.7.0",
     "fetch": "^2.0.2",
-    "file-saver": "^1.3.3"
+    "file-saver": "^1.3.3",
+    "vimeo-player-js": "^2.1.0"
```

Comment:
Do we have a good way to prevent this from being loaded for every single page? Do we stick it in Vendor for that? Could also load it via their CDN on this page only.